scripts/validation/validate.js

```diff
@@ -27,16 +27,13 @@ if (args.includes('--schema-only')) {
   args.splice(args.indexOf('--schema-only'), 1);
 }
 
-const modifiedOnly = args.includes('--modified-only');
 let servicesToValidate = args;
 
 (async () => {
   const SERVICE_DECLARATIONS_PATH = services.getServicesDeclarationsPath();
   const serviceDeclarations = await services.loadWithHistory();
 
-  if (modifiedOnly) {
-    servicesToValidate = await services.getIdsOfModified();
-  } else if (!servicesToValidate.length) {
+  if (!servicesToValidate.length) {
```

Comment:
This should be replaced with the possibility to validate one service only

Like "npm run validate Amazon"

If we do not have this, it will be very difficult to add a service and validate it without running the whole test suite

Reply:
It's already the case :wink: